pumpkin-lib/src/engine/constraint_satisfaction_solver.rs

```diff
@@ -469,10 +473,10 @@ impl ConstraintSatisfactionSolver {
     ///
     /// *Notes:*
     ///   - If the solver is not in an unsatisfied state, this method will panic.
-    ///   - If the solver is in an unsatisfied state, but solving was done without assumptions,
-    ///   this will return an empty vector.
-    ///   - If the assumptions are inconsistent, i.e. both literal x and !x are assumed, an error
-    ///   is returned, with the literal being one of the inconsistent assumptions.
+    ///   - If the solver is in an unsatisfied state, but solving was done without assumptions, this
```

Comment:
why does the formating change in this case? The old version seemed better to me, since the new version breaks up the second line in an awkward spot. Similar changes are also in other places.